qutebrowser/config/configdata.py

```diff
@@ -321,6 +321,10 @@ def data(readonly=False):
              SettingValue(typ.String(none_ok=True), 'en-US,en'),
              "Value to send in the `accept-language` header."),
 
+            ('referer-header',
+             SettingValue(typ.Referer(), 'same-domain'),
+             "Send the Referer header"),
```

Comment:
It still bugs me this was misspelled in the standard and now the wrong spelling is the commonly used one :wink: